pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -86,6 +91,133 @@ func NewCIDataClient(dataCoordinates BigQueryDataCoordinates, client *bigquery.C
 	}
 }
 
+func (c *ciDataClient) ListDisruptionHistoricalData(ctx context.Context) ([]jobrunaggregatorapi.HistoricalDataRow, error) {
+	return c.listDisruptionHistoricalData(ctx)
```

Comment:
Nit: Indirection here probably not needed, might be worth putting the code in here instead.

Reply:
Sure thing, I was just keeping it inline with some of the other functions I saw